packages/rng_tools.rb

```diff
@@ -0,0 +1,76 @@
+require 'package'
+
+class Rng_tools < Package
+  description 'This is a random number generator daemon.'
+  homepage 'https://github.com/nhorman/rng-tools'
+  version '6.5'
+  source_url 'https://github.com/nhorman/rng-tools/archive/v6.5.tar.gz'
+  source_sha256 '5fecd904f7d01262b3209ff78dd3b9594aac8daa41badd5a1e6438658e80c36e'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'curl'
+  depends_on 'libgcrypt'
+  depends_on 'sysfsutils'
+  depends_on 'psmisc'
+
+  def self.build
+    system './autogen.sh'
+    system "./configure", "--prefix=#{CREW_PREFIX}", "--libdir=#{CREW_LIB_PREFIX}"
+    system 'make'
+    system "echo '#!/bin/bash' > startrngd"
+    system "echo 'RNGD=#{CREW_PREFIX}/sbin/rngd' >> startrngd"
+    system "echo 'RNG=\$(pidof \$RNGD 2> /dev/null)' >> startrngd"
+    system "echo 'if [ -z \"\$RNG\" ]; then' >> startrngd"
+    system "echo '  sudo LD_LIBRARY_PATH=\$LD_LIBRARY_PATH rngd &>/dev/null' >> startrngd"
```

Comment:
Not sure LD_LIBRARY_PATH is necessary.  Should already be set.

Reply:
It complains about a missing `libsysfs.so.2`. Not much I can do there.